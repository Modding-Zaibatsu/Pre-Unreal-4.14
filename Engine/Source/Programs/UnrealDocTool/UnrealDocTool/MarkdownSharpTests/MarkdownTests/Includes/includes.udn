
[RELATIVE:Temp\Images\__ctor]

[INCLUDE:Temp\Images]

[INCLUDE:Temp\Images#Excerpt(offset:1)]

[TOC(start:2 end:3)]

[TOC]

%Temp\Images:toc%

[INCLUDE:Temp\Images#doxyInclusionTesting]