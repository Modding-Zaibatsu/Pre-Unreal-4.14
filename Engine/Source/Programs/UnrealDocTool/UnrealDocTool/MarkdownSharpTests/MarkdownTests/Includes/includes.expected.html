<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
    <head>
    </head>
    <body>
        Images\__ctor\index.html
        
        <a name="-192392570" ></a>
        <h1 id="1stopheaderlevel1">1. Header Level 1</h1>
        <h2 id="astopheaderlevel2">a. Header Level 2</h2>
        <h2 id="bstopheaderlevel2">b. Header Level 2</h2>
        <h1 id="2stopheaderlevel1">2. Header Level 1</h1>
        <h1 id="3stopexcerptheaderlevel1">3. Excerpt header Level 1</h1>
        <h2 id="astopexcerptheaderlevel2">a. Excerpt header Level 2</h2>
        <h3 id="-stopexcerptheaderlevel3">-. Excerpt header Level 3</h3>
        <h2 id="3stopexcerptheaderlevel1">3. Excerpt header Level 1</h2>
        <h3 id="astopexcerptheaderlevel2">a. Excerpt header Level 2</h3>
        <h4 id="-stopexcerptheaderlevel3">-. Excerpt header Level 3</h4>
        <ul>
            <li><a href="#astopheaderlevel2">a. Header Level 2</a></li>
            <li><a href="#bstopheaderlevel2">b. Header Level 2</a></li>
            <li><a href="#astopexcerptheaderlevel2">a. Excerpt header Level 2</a></li>
            <ul>
                <li><a href="#-stopexcerptheaderlevel3">-. Excerpt header Level 3</a></li>
            </ul>
            <li><a href="#3stopexcerptheaderlevel1">3. Excerpt header Level 1</a></li>
            <ul>
                <li><a href="#astopexcerptheaderlevel2">a. Excerpt header Level 2</a></li>
            </ul>
        </ul>
        <ul>
            <li><a href="#1stopheaderlevel1">1. Header Level 1</a></li>
            <ul>
                <li><a href="#astopheaderlevel2">a. Header Level 2</a></li>
                <li><a href="#bstopheaderlevel2">b. Header Level 2</a></li>
            </ul>
            <li><a href="#2stopheaderlevel1">2. Header Level 1</a></li>
            <li><a href="#3stopexcerptheaderlevel1">3. Excerpt header Level 1</a></li>
            <ul>
                <li><a href="#astopexcerptheaderlevel2">a. Excerpt header Level 2</a></li>
                <ul>
                    <li><a href="#-stopexcerptheaderlevel3">-. Excerpt header Level 3</a></li>
                </ul>
                <li><a href="#3stopexcerptheaderlevel1">3. Excerpt header Level 1</a></li>
                <ul>
                    <li><a href="#astopexcerptheaderlevel2">a. Excerpt header Level 2</a></li>
                    <ul>
                        <li><a href="#-stopexcerptheaderlevel3">-. Excerpt header Level 3</a></li>
                    </ul>
                </ul>
            </ul>
        </ul>
        <ul>
            <li>
                <a href="./../../INT\Temp\Images\index.html#H1TitleId">Included file</a>
                <ul>
                    <li><a href="./../../INT\Temp\Images\index.html#1stopheaderlevel1">1. Header Level 1</a></li>
                    <ul>
                        <li><a href="./../../INT\Temp\Images\index.html#astopheaderlevel2">a. Header Level 2</a></li>
                        <li><a href="./../../INT\Temp\Images\index.html#bstopheaderlevel2">b. Header Level 2</a></li>
                    </ul>
                    <li><a href="./../../INT\Temp\Images\index.html#2stopheaderlevel1">2. Header Level 1</a></li>
                    <li><a href="./../../INT\Temp\Images\index.html#3stopexcerptheaderlevel1">3. Excerpt header Level 1</a></li>
                    <ul>
                        <li><a href="./../../INT\Temp\Images\index.html#astopexcerptheaderlevel2">a. Excerpt header Level 2</a></li>
                        <ul>
                            <li><a href="./../../INT\Temp\Images\index.html#-stopexcerptheaderlevel3">-. Excerpt header Level 3</a></li>
                        </ul>
                    </ul>
                </ul>
            </li>
        </ul>
        
        <p>First sentence. The rest of the comment.</p>
    </body>
</html>
